calico_node/startup/startup.go

```diff
@@ -497,23 +513,25 @@ func configureIPPools(client *client.Client) {
 	// Ensure there are pools created for each IP version.
 	if !ipv4Present {
 		log.Debug("Create default IPv4 IP pool")
-		createIPPool(client, ipv4Cidr, ipv4IpipModeEnvVar)
+		OutgoingNATisDefined, OutgoingNATisValue := evaluateENVBool("CALICO_IPV4POOL_NAT_OUTGOING", "NAT_Outgoing disabled for Default IPv4 Pool.")
+		createIPPool(client, ipv4Cidr, ipv4IpipModeEnvVar, OutgoingNATisDefined, OutgoingNATisValue)
 	}
 	if !ipv6Present && ipv6Supported() {
 		log.Debug("Create default IPv6 IP pool")
-		createIPPool(client, ipv6Cidr, string(ipip.Undefined))
+		OutgoingNATisDefined, OutgoingNATisValue := evaluateENVBool("CALICO_IPV6POOL_NAT_OUTGOING", "NAT_Outgoing disabled for Default IPv6 Pool.")
+		createIPPool(client, ipv6Cidr, string(ipip.Undefined), OutgoingNATisDefined, OutgoingNATisValue)
 	}
 }
 
+
 // ipv6Supported returns true if IPv6 is supported on this platform.  This performs
 // a check on the appropriate Felix parameter and if supported also performs a
 // simplistic check of /proc/sys/net/ipv6 (since platforms that do not have IPv6
 // compiled in will not have this entry).
 func ipv6Supported() bool {
-	// First check the Felix parm.
-	switch strings.ToLower(os.Getenv("FELIX_IPV6SUPPORT")) {
-	case "false", "0", "no", "n", "f":
-		log.Info("IPv6 support disabled through environment")
+	// First check if Felix param is false
+	_, isValue := evaluateENVBool("FELIX_IPV6SUPPORT", "IPv6 support disabled through environment");
```

Comment:
I think whatever you change isValue to in evaluateENVBool you can probably use that name here too.